manager-rs/dauth-service/src/manager.rs

```diff
@@ -24,7 +25,21 @@ pub async fn find_vector(
     if let Ok(vector) = generate_auth_vector(context.clone(), av_request).await {
         Ok(vector)
     } else {
-        todo!()
+        let (home_network_id, backup_network_ids) = clients::directory::lookup_user(context.clone(), &av_request.user_id).await?;
+
+        let (home_address, _) = clients::directory::lookup_network(context.clone(), &home_network_id).await?;
+        if let Ok(vector) = clients::home_network::get_auth_vector(context.clone(), &av_request.user_id, &home_address).await {
+            Ok(vector)
+        } else {
```

Comment:
Minor non-blocking style feedback, that if you're doing a series of cases, often time's it clearer to write in a flat style with early returns at the end of each case, rather than nesting inside an else. For 3 cases the nesting isn't too terrible, but it gets rough as the number of cases grows :slightly_smiling_face:

Reply:
Fair point! I'll look at changing that going forward